ArchiSteamFarm/Steam/Data/Confirmation.cs

```diff
@@ -35,6 +35,11 @@ public sealed class Confirmation {
 	[JsonRequired]
 	public EConfirmationType ConfirmationType { get; private init; }
 
+	[JsonInclude]
+	[JsonPropertyName("type_text")]
+	[JsonRequired]
+	public string ConfirmationTypeText { get; private init; } = null!;
```

Comment:
Nit: try to avoid `null!` if possible. In some cases you need to, e.g. because constructor is private or you don't have a sane default value to initialize this with, e.g. `Uri`, but in all other cases it makes sense to provide default value, such as empty array, empty string, or `new()` :slightly_smiling_face: